src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -215,10 +217,14 @@ class AuthScreens extends React.Component {
         };
         const fullscreenModalScreenOptions = {
             ...commonModalScreenOptions,
-            cardStyle: {...styles.fullscreenCard},
+            cardStyle: {
+                ...styles.fullscreenCard,
```

Comment:
Since `styles.cardOverlay` is being added below and has the following styles:
```js
    cardOverlay: {
        backgroundColor: themeColors.modalBackdrop,
        position: 'absolute',
        top: 0,
        left: 0,
        width: '100%',
        height: '100%',
        opacity: 0.5,
    }
```

and `styles.fullscreenCard` has the following styles:
```js
    fullscreenCard: {
        position: 'absolute',
        left: 0,
        top: 0,
        width: '100%',
        height: '100%',
    }
```

why do we need to add `...styles.fullscreenCard` here? Won't all these styles be included in `cardOverlay`?

Reply:
The `cardOverlay` is for the backdrop of the modal, and the `fullscreenCard` is for the content that appears on top of the backdrop. These styles are for two different components, hence the need for the two different styles.

Reply:
I was just wondering if the styles would be inherited. In any case, if the `cardOverlay` component might be used elsewhere, it makes sense to pass the styles along and not rely on inheritence.